include/transformations/io-transformations/IOUtils.h

```diff
@@ -0,0 +1,14 @@
+#ifndef INCLUDE_TRANSFORMATIONS_IO_TRANSFORMATIONS_IOUTILS_H_
```

Comment:
Why do you call it `IOUtils`? For now, there are no functions for working with input and output. Only some string manipulation.